### ISSUE TITLE
Open places displaying as "Closed"

### ISSUE THREAD

**Issue by Louie-lasse** [cthit/mat#97]
The following places show as "Closed" (at friday 12:30), desplite beeing open according to Google Maps AND mat.chalmers.it
J.A. Pripps
Johanneberg Sushi

**Comment by Portals** [comment]
Yeah seems like the times we set earlier have been updated. Probably "covid-19" times. Will fix.
